types/LinkEventTypes.d.ts

```diff
@@ -0,0 +1,54 @@
+export type LinkEventListener = (linkEvent: LinkEvent) => void
+
+ 
+export interface LinkEvent {
+  eventName: LinkEventName;
+  metadata: LinkEventMetadata;
+}
+ 
+export interface LinkEventMetadata {
+  error_code: string;
+  error_message: string;
+  error_type: string;
+  exit_status: string;
+  institution_id: string;
+  institution_name: string;
+  institution_search_query: string;
+  request_id: string;
+  link_session_id: string;
+  mfa_type: string;
+  view_name: LinkEventViewName;
+  timestamp: string;
```

Comment:
Order should be:

1. sessionID
2. mfatype
3. request id
4. timestamp
5. view name